### PR TITLE
October

### PR DESCRIPTION
**Pull request by bibash28**
## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [x] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: lib/activity_log/log_class.dart [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import 'package:altme/activity_log/activity_log.dart';
+import 'package:altme/app/app.dart';
```
Comment (hawkbee1) [inline lib/activity_log/log_class.dart:2]: I don't think you need this import. I think I removed it from main while handling some chores

File: lib/app/shared/constants/constants_json.dart [hunk 2]
```diff
@@ -324,7 +324,7 @@ abstract class ConstantsJson {
           'JsonWebSignature2020',
           'Ed25519Signature2018',
           'EcdsaSecp256k1Signature2019',
-          'RsaSignature2018',
```
Comment (hawkbee1) [inline lib/app/shared/constants/constants_json.dart:327]: you should have a warning for this in vscode.

File: lib/onboarding/gen_phrase/cubit/onboarding_gen_phrase_cubit.dart [hunk 3]
```diff
@@ -41,7 +41,10 @@ class OnBoardingGenPhraseCubit extends Cubit<OnBoardingGenPhraseState> {
 
   final log = getLogger('OnBoardingGenPhraseCubit');
 
-  Future<void> generateSSIAndCryptoAccount(List<String> mnemonic) async {
+  Future<void> generateSSIAndCryptoAccount({
+    required List<String> mnemonic,
+    required bool restoreWallet,
```
Comment (hawkbee1) [inline lib/onboarding/gen_phrase/cubit/onboarding_gen_phrase_cubit.dart:46]: wallet restoration is not the purpose of generateSSIAndCryptoAccount. It should be handled in separate process

**Review by hawkbee1 — CHANGES_REQUESTED** [review]
A few comments

File: lib/activity_log/log_class.dart (hunk 1 above)
Comment (bibash28) [reply lib/activity_log/log_class.dart]: yes.

File: lib/app/shared/constants/constants_json.dart (hunk 2 above)
Comment (bibash28) [reply lib/app/shared/constants/constants_json.dart]: yes. I got the warning.

File: lib/onboarding/gen_phrase/cubit/onboarding_gen_phrase_cubit.dart (hunk 3 above)
Comment (bibash28) [reply lib/onboarding/gen_phrase/cubit/onboarding_gen_phrase_cubit.dart]: `generateSSIAndCryptoAccount` is not called during the restoration process.

**Pull request merged**